### PR TITLE
refactor: added missing hover and unhover methods to simulator

### PR DESCRIPTION
**Pull request by SimonTheSourcerer**
### Description
Needed for desktop mode.

### CODE REVIEW

File: Runtime/Interaction/InteractionSimulatorDummy.cs [hunk 1]
```diff
@@ -45,7 +45,17 @@ public override void StopUse(IInteractableObject interactable)
...
             Debug.LogWarning(ErrorMessage);
         }
-        
+
+        public override void HoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
+        {
+            Debug.LogWarning(ErrorMessage);
+        }
+
+        public override void UnhoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
```
Comment (Gusinuhe) [inline Runtime/Interaction/InteractionSimulatorDummy.cs:54]: Missing docs

**Review by tomwim — DISMISSED** [review]

File: Runtime/Interaction/InteractionSimulatorDummy.cs [hunk 2]
```diff
@@ -45,7 +45,17 @@ public override void StopUse(IInteractableObject interactable)
         {
             Debug.LogWarning(ErrorMessage);
         }
-        
+
+        public override void HoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
+        {
+            Debug.LogWarning(ErrorMessage);
+        }
+
```
Comment (SimonTheSourcerer) [inline Runtime/Interaction/InteractionSimulatorDummy.cs:53]:
```suggestion

/// <inheritdoc />
```

File: Runtime/Interaction/InteractionSimulatorDummy.cs [hunk 3]
```diff
@@ -45,7 +45,17 @@ public override void StopUse(IInteractableObject interactable)
         {
             Debug.LogWarning(ErrorMessage);
         }
-        
+
```
Comment (SimonTheSourcerer) [inline Runtime/Interaction/InteractionSimulatorDummy.cs:48]:
```suggestion

/// <inheritdoc />
```

File: Runtime/Interaction/InteractionSimulatorDummy.cs [hunk 4]
```diff
@@ -45,7 +45,21 @@ public override void StopUse(IInteractableObject interactable)
...
+        /// <inheritdoc />
+/// <inheritdoc />
+        public override void HoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
```
Comment (SimonTheSourcerer) [inline Runtime/Interaction/InteractionSimulatorDummy.cs:51]:
```suggestion
        /// <inheritdoc />
        public override void HoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
```

File: Runtime/Interaction/InteractionSimulatorDummy.cs [hunk 5]
```diff
@@ -45,7 +45,21 @@ public override void StopUse(IInteractableObject interactable)
...
+        {
+            Debug.LogWarning(ErrorMessage);
+        }
+
+        /// <inheritdoc />
+/// <inheritdoc />
+        public override void UnhoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
```
Comment (SimonTheSourcerer) [inline Runtime/Interaction/InteractionSimulatorDummy.cs:58]:
```suggestion
        /// <inheritdoc />
        public override void UnhoverSnapZone(ISnapZone snapZone, IInteractableObject interactable)
```

**Review by Gusinuhe — APPROVED** [review]

**Review by tomwim — APPROVED** [review]

**Pull request merged**